### DISCUSSION TITLE
Faction names

### DISCUSSION THREAD

**Discussion by BluFox2003** [official-antistasi-community/A3-Antistasi#2190] [Q&A] [answered]
Hey, just wondering what the names of the factions are saved as, e.g. what would the 3CB AAF template save as. for example the standard blufor factions classname is BLU_F
sorry if this is a confusing question
Thanks!

**Comment by Bob-Murphy** [comment, marked as answer]
If you for example look at the template folder for 3CB (https://github.com/official-antistasi-community/A3-Antistasi/tree/unstable/A3-Antistasi/Templates/Templates/3CB) and then look at the file naming like for example 3CN_AI_AAF.sqf, I guess that you will find what you search.
